ballista/rust/core/src/utils.rs

```diff
@@ -316,38 +313,3 @@ impl<T: 'static + AsLogicalPlan> QueryPlanner for BallistaQueryPlanner<T> {
         }
     }
 }
-
-pub struct WrappedStream {
```

Comment:
This has been replaced by RecordBatchStreamAdapter which also does not require boxing or pinning the stream, making it not only easier to use, but potentially slightly faster